html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-block-and-inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>
+      Part of our new Americana collection, this <i>beautiful</i> blue is
+      already a customer favorite!
+    </p>
+    <p>Enjoy complete <b>Freedom of Movement!</b></p>
```

Comment:
We want to use `strong` over `b`. This is because `strong` is a semantic element. We will touch on this in Q&A